### DISCUSSION TITLE
Add new users to the Org their Team is in

### DISCUSSION THREAD

**Discussion by techgnosis** [grafana/grafana#42692] [General]
Current State - When you add a user to Grafana, either via the UI or via a Team with an external Group, the user is added to Org 1 regardless if the team is in Org 1 or not. 

Ideal Future State - When you add a user to Grafana, either via the UI or via a Team with an external Group, the user is added to the Org that the Team belongs to.

This issue exists whether `auto_assign_org` is `true` or `false`

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
